src/test/java/net/sf/jabref/logic/cleanup/CleanupWorkerTest.java

```diff
@@ -323,4 +327,10 @@ public void cleanupWithDisabledFieldFormatterChangesNothing() {
         worker.cleanup(preset, entry);
         Assert.assertEquals(Optional.of("01"), entry.getField("month"));
     }
+
+    @After
+    public void tearDown() {
+        JabRefPreferences.getInstance().putBoolean(JabRefPreferences.BIB_LOC_AS_PRIMARY_DIR, false);
+
```

Comment:
Would be cool if you can remove all thos obsolete empty lines 😄 I know its not important but 😸